hieradata/dev/wso2/wso2is/pattern-1/default.yaml

```diff
@@ -53,11 +52,14 @@ wso2::template_list:
 #     http: 80
 #     https: 443
 
-# Puppet file list to be populated
+# Puppet file list to be copied triggering restart
```

Comment:
Please refer feedback in https://github.com/wso2/puppet-base/pull/33 for this comment line.